src/sentry/tasks/relay.py

```diff
@@ -87,6 +101,11 @@ def schedule_build_project_config(public_key):
         public_key=public_key, project_id=None, organization_id=None
     )
 
+    # TODO: Temprorary task
+    check_build_project_config.apply_async(
+        kwargs={"public_key": public_key, "tmp_scheduled": tmp_scheduled}, countdown=7
```

Comment:
What is the purpose of adding `countdown` here? (starting another thread since it's a different topic).

Reply:
countdown delays task by 7 seconds before it runs

Reply:
And what's the purpose of doing that, I guess to run `check_build_project_config` after `build_project_config`?

Reply:
yeah exactly. you could also spawn the check task inside build_project_config, but the idea is to find dropped/crashed build-tasks